userspace/libscap/CMakeLists.txt

```diff
@@ -146,3 +141,24 @@ if(CREATE_TEST_TARGETS AND NOT WIN32)
 		# Add unit test directories
 		add_subdirectory(test)
 endif()
+
+if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+	add_subdirectory(linux)
```

Comment:
Couldn't the empty CMakeLists under linux/ and win32/ do
`include_directories(${CMAKE_CURRENT_SOURCE_DIR)` ?
We could avoid the include_directories below, and it makes more sense than having empty cmakelists.

Reply:
Makes sense, even though the platform dirs will have some .c files eventually :)